src/_blog/libp2p-and-Ethereum.md

```diff
@@ -0,0 +1,311 @@
+There, [David Dias](https://research.protocol.ai/authors/david-dias/) gave a talk on September 2016 titled "[libp2p ❤ devp2p: IPFS and Ethereum Networking](https://archive.devcon.org/archive/watch/2/libp2p-devp2p-ipfs-and-ethereum-networking/?tab=YouTube)".
+
+In this talk David gave an overview of libp2p and gave a demo. He ran the EVM in a browser and modified a go-ethereum/Geth node to use libp2p. The Geth node running libp2p connected to the Ethereum mainnet, and the EVM nodes in the browsers (also running libp2p) were able to fetch blocks from the Geth node.
+At the time, this was a big breakthrough and a sign of things to come.
+
+Following this presentation, Protocol Labs and Ethereum [continued working together](https://github.com/ipfs/notes/issues/173) to see how it would be possible to integrate libp2p.
+
+<div class="container" style="display:flex; column-gap:10px;">
+    <figure>
+        <img src="https://i.imgur.com/hfy2O8S.png"/>
+    </figure>
+</div>
+
+Conversations continued into Devcon 3 in 2017.
+There Felix gave a [talk on devp2p](https://archive.devcon.org/archive/watch/3/evolving-devp2p/?tab=YouTube) which mentioned integration options with libp2p.
+Following the event, he published the "[Ethereum ♥ libp2p](https://twurst.com/articles/eth-loves-libp2p.html)" article that expanded on what a libp2p integration could look like:
+
+*"I've been thinking about ways to integrate libp2p for quite a while...My initial goal was to join their DHT, which turned out to be impractical. There's a much easier thing we can do though: use libp2p's transports and multiplexing algorithm. This saves us a bunch of code and gives us a few new protocols...that we can try out without too much work...The ultimate goal is to adopt libp2p's transport protocols as the way Ethereum nodes communicate.*"
+
+This showcased the desire by both communities to leverage the benefits of libp2p.
+
+##  Designing libp2p into the Beacon Chain (2018-2019) :writing_hand: 
+The above discussions were about integrating libp2p into the Ethereum mainnet, either replacing or augmenting devp2p.
+
+In 2018, the Ethereum community put major focus on how to scale the network. As the year progressed, the plan became to deliver both scalability and proof-of-stake as a part of "Ethereum 2.0" (for more information, see: [State of the Ethereum Protocol #1](https://media.consensys.net/state-of-ethereum-protocol-1-d3211dd0f6)).
+Both solutions were to be developed on a separate chain from the mainnet, called the [Beacon Chain](https://ethereum.org/en/upgrades/beacon-chain/).
+
+At this time, the Ethereum 2.0 community began evaluating the networking requirements for the Beacon Chain. Designing a libp2p integration became a formal effort. Focus moved towards adding libp2p to the Beacon Chain instead of extending/replacing devp2p on the Ethereum mainnet.
```

Comment:
```suggestion
At this time, the Ethereum 2.0 community began evaluating the networking requirements for the Beacon Chain. Focus moved towards adding libp2p to the Beacon Chain instead of extending or replacing devp2p on the Ethereum mainnet. 

Designing a libp2p integration became a formal effort.
```